android/src/main/java/com/it_nomads/fluttersecurestorage/FlutterSecureStoragePlugin.java

```diff
@@ -59,6 +59,13 @@ public void onMethodCall(MethodCall call, Result result) {
                     result.success(null);
                     break;
                 }
+                case "contains": {
```

Comment:
Could you please add implementation for iOS plugin?

Reply:
Sorry. I don't have the development environment for IOS.